src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,91 @@
+import "./Pagination.css";
+import React from "react";
+import { useState } from "react";
+import LoadList from "../LoadList/LoadList";
+
+const Pagination = ({
+  data,
+  pageLimit,
+  dataLimit,
+  showDelete,
+  showInformation,
+}) => {
+  const pages = Math.round(data.length / dataLimit);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  function goToNextPage() {
+    setCurrentPage((page) => page + 1);
+  }
+
+  function goToPreviousPage() {
+    setCurrentPage((page) => page - 1);
+  }
+
+  function changePage(event) {
+    const pageNumber = Number(event.target.textContent);
+    setCurrentPage(pageNumber);
+  }
+
+  const getPaginatedData = () => {
+    const startIndex = currentPage * dataLimit - dataLimit;
+    const endIndex = startIndex + dataLimit;
+    return data.slice(startIndex, endIndex);
+  };
+
+  const getPaginationGroup = () => {
+    let start = Math.floor((currentPage - 1) / pageLimit) * pageLimit;
+    return new Array(pageLimit).fill().map((_, idx) => start + idx + 1);
+  };
+
+  return (
+    <>
+      <>
+        <LoadList
```

Comment:
This might require some refactoring and some re-thinking of your logic, but it's best if the Robot list didn't sit inside this component. Your top level should be something more like:
`Header`
`List`
`Pagination`